guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,140 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
+
+.Prerequisites
+
+* Ensure that you meet the requirements for HTTP booting. For more information, see link:{BaseURL}/planning_for_red_hat_satellite_6/chap-red_hat_satellite-architecture_guide-provisioning_concepts#http_booting_requirements[HTTP Booting Requirements] in _Planning for {Project}_.
+
+.Procedure
+
+. On {Project}, update the `grub2-efi` package to the latest version:
```

Comment:
This needs to be done on (HTTPBoot) Capsule(s). It can be the embedded Capsule (thus Server).